humans/tasks.py

```diff
@@ -72,6 +72,12 @@ def update_public_keys():
             user.keyserver_url = ""
             user.save()
         elif state == "valid":
+            # Warns user if key about to expire
```

Comment:
This should be a separate task since like you can see on line 38, this task only checks/updates the users that defined the a `keyserver_url`. This feature should be available for everyone.